instrumentation/spring-webmvc-3.1/spring-webmvc-3.1.gradle

```diff
@@ -5,7 +5,11 @@ muzzle {
     group = 'org.springframework'
     module = 'spring-webmvc'
     versions = "[3.1.0.RELEASE,]"
-    skipVersions += ['1.2.1', '1.2.2', '1.2.3', '1.2.4'] // broken releases... missing dependencies
+    skipVersions += [
```

Comment:
Not sure new format is better :)

Reply:
yeah, I agree... Intellij does a better job at formatting these, but unfortunately doesn't have a command line tool, so no validation of proper formatting.  If you have a better configuration please suggest.

Reply:
I would lean towards not auto formatting Gradle files if the formatter doesn't give a result we're happy with, they're a very small part of this repo, compared to Java.

Reply:
reverted auto-formatting of gradle files for now